webrtc/RTCStats-helper.js

```diff
@@ -145,42 +149,29 @@ function validateRtcStats(statsReport, stats) {
   [webrtc-stats]
   7.1.  RTCRTPStreamStats dictionary
     dictionary RTCRTPStreamStats : RTCStats {
```

Comment:
nit: RTCRTPStreamStats has been renamed RTCRtpStreamStats and the same with RTCRececeivedRtpStreamStats, RTCSentRtpStreamStats, RTCOutboundRtpStreamStats, RTCInboundRtpStreamStats, RTCRemoteOutboundRtpStreamStats and RTCRemoteInboundRtpStreamStats. You may optionally update the casing in this PR to reflect that.